test/commons/dom/visually-contains.js

```diff
@@ -47,4 +47,12 @@ describe('dom.visuallyContains', function () {
 		assert.isTrue(axe.commons.dom.visuallyContains(target, target.parentNode));
 	});
 
+	it('should return true when element is inline', function () {
```

Comment:
I can add some more tests for other scenarios, but this was definitely a problem case.